docker/etl/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# syntax=docker/dockerfile:1
+FROM python:3.10.7
+WORKDIR /code
+
+# install nix for data extraction
+RUN curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install linux --init none --no-confirm
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV POETRY_VIRTUALENVS_CREATE=0
+ENV PATH="${PATH}:/nix/var/nix/profiles/default/bin"
+ENV PATH="/venv/bin:${PATH}"
+ENV VIRTUAL_ENV="/venv"
+
+# if you use an alpine python image, the following is necessary
+# RUN apk add --no-cache gcc musl-dev linux-headers
+# Note: This version of poetry must match what is in the `flake.nix`
+RUN pip install "poetry==1.5.1"
+
+RUN python -m venv /venv
```

Comment:
We don't need a venv in this sort of container
```suggestion
```

I'm not sure poetry picks up `VIRTUAL_ENV` so this could potentially break thing (idk)

Reply:
I see your point. 
Since we don't need a venv, I removed `VIRTUAL_ENV` and other commands including venv, and it worked well.